engine/src/main/java/org/terasology/rendering/dag/nodes/OpaqueBlocksNode.java

```diff
@@ -48,41 +48,31 @@
  * In a typical world this is the majority of the world's landscape.
  */
 public class OpaqueBlocksNode extends AbstractNode implements WireframeCapable {
-    private static final ResourceUrn CHUNK_SHADER = new ResourceUrn("engine:prog.chunk");
+    private static final ResourceUrn CHUNK_MATERIAL = new ResourceUrn("engine:prog.chunk");
 
-    @In
     private WorldRenderer worldRenderer;
-
-    @In
-    private Config config;
-
-    @In
     private RenderQueuesHelper renderQueues;
 
-    @In
-    private DisplayResolutionDependentFBOs displayResolutionDependentFBOs;
-
     private Camera playerCamera;
     private Material chunkShader;
     private SetWireframe wireframeStateChange;
     private RenderingDebugConfig renderingDebugConfig;
 
-    /**
-     * Initialises this node. -Must- be called once after instantiation.
-     */
-    @Override
-    public void initialise() {
+    public OpaqueBlocksNode(Context context) {
+        renderQueues = context.get(RenderQueuesHelper.class);
+        worldRenderer = context.get(WorldRenderer.class);
+
         wireframeStateChange = new SetWireframe(true);
-        renderingDebugConfig = config.getRendering().getDebug();
+        renderingDebugConfig = context.get(Config.class).getRendering().getDebug();
```

Comment:
Wherever possible, I've tried to go for the one-line code as per your previous comment. However, in very few cases (not sure if Context was one of these cases) other lines used the varaible too, so I went for the "get variable in one line, use in other lines" approach to avoid re-fetching.

Also, the renderingDebugConfig is used in a lot of Nodes, and is probably the only one that gets (slightly) confusing. All the other one-line codes are pretty readable. If you want, I can convert this to two lines.. although I believe we can leave this as it is, since `config.getRendering().getDebug()` isn't much clearer either..